sabnzbd/assembler.py

```diff
@@ -169,6 +169,11 @@ def _assemble(nzf, path, dupe):
     decodetable = nzf.decodetable
 
     for articlenum in decodetable:
+        # Break if deleted during writing
+        if nzf.nzo.deleted:
```

Comment:
It's the same that we use a few lines higher (after this function)
https://github.com/sabnzbd/sabnzbd/pull/658/files#diff-3ffdc5044910fc8e1d1913955f7ee55fR91